src/charts/hover.js

```diff
@@ -124,10 +130,12 @@ export default function Hover (_container) {
     })
 
     const dotsStack = data.stack([stackedDataPoint])
+    const leftAxisGroup = data.groupKeys[LEFT_AXIS_GROUP_INDEX]
     const dotsData = dotsStack.map((d) => {
       const dot = {}
       dot.value = d[0][1]
       dot.id = d.key
+      dot.group = leftAxisGroup[0] // set to 1st left axis group as stacked is always single axis
```

Comment:
This was causing the hover dot to use the wrong scale when stack type, which was not really visible since stack always forces single axis in Immerse.